.github/workflows/run-composition.yml

```diff
@@ -56,17 +62,23 @@ jobs:
           echo "::set-output name=custom_git_sha::${SHA}"
       - name: Build the composition file
         working-directory: ./test-plans
+        timeout-minutes: 30
         run: |
-          testground build composition                        \
-            -f ${{ inputs.composition_file }}                 \
-            --wait
+          for i in 1 2 3; do
+            echo "=== Attempt $i ==="
+            testground build composition                        \
+              -f ${{ inputs.composition_file }}                 \
+              --wait && exit 0;
+            sleep 10
```

Comment:
Why do we need to sleep in between retries?

Reply:
The retry might happen almost instantaneously because of the docker caching. The goal here is to wait some time to give remote services a chance to improve the situation (like the 404 above).

I'll merge as is, but there will be at least one follow-up PR for improvements, where we can remove this sleep if we don't want it.